frontend/src/shared/components/header/Header.tsx

```diff
@@ -119,7 +126,8 @@ export const Header = ({ routeBreadCrumb }: HeaderProps) => {
             shadow='lg'
             overflowY='scroll'
             p={15}
-            maxH={500}
+            maxH={800}
```

Comment:
Can consider changing the Notif container constraints to `maxH="80vh"` so that it dynamically adapts to smaller screens?

![pxMRNtIW0d](https://user-images.githubusercontent.com/90462635/194788189-26acd0dd-47ff-4dc6-9f20-3f36066fd404.png)

Reply:
this is noted. will be implementing this before merging